stable/pipelines/values.yaml

```diff
@@ -617,6 +625,9 @@ pipelines:
       defaultMinionInstanceSize: "c4.large"
       allowDynamicNodes: true
       allowCustomNodes: true
+      {{- range $key, $value := .Values.runtimeOverride }}
```

Comment:
Wouldn't it be easier here to just use `toYaml`?

Reply:
I copied what @khazina had done

Reply:
Should probably be `with` and `tpl`. We can change it later on...